cypress/e2e/acceptance/birth/birth-search.cy.js

```diff
@@ -76,6 +76,17 @@ describe('Birth search', () => {
       BirthSearchPage.noForenames();
     });
   });
+  describe('with a first name more than 30 character length', () => {
+    before(() => {
+      BirthSearchPage.visit();
+      BirthSearchPage.performSearch({
+        surname: 'Surname', forenames: 'forenamemorethanthirtycharacter', day: '01', month: '10', year: '2010'
```

Comment:
I think this is more a cosmetic comment, but could we instead use Pascal or camelCase for the `forenames` attribute?

Reply:
you mean to use string forenamemorethanthirtycharacter -> forenameMoreThanThirtyCharacter, I thought it was a name so wasn't sure if we want it like this to pass in the name field